staging/src/k8s.io/kubectl/pkg/describe/describe.go

```diff
@@ -1667,6 +1667,10 @@ func getPodsForPVC(c corev1client.PodInterface, pvcName string, settings Describ
 		for _, volume := range pod.Spec.Volumes {
 			if volume.VolumeSource.PersistentVolumeClaim != nil && volume.VolumeSource.PersistentVolumeClaim.ClaimName == pvcName {
 				pods = append(pods, pod)
+				continue
+			}
+			if volume.Ephemeral != nil && volume.Ephemeral.VolumeClaimTemplate != nil && pod.Name+"-"+volume.Name == pvcName {
```

Comment:
Why do we do this generation `pod.Name+"-"+volume.Name`?. Is it always equal to `pvcName` and could you please give the reference of this generation in code base?

Reply:
A better way is to reuse the [VolumeIsForPod](https://github.com/kubernetes/kubernetes/blob/master/staging/src/k8s.io/component-helpers/storage/ephemeral/ephemeral.go)